### PR TITLE
CSHARP-2770: Update build script for FLE

### PR DESCRIPTION
**Pull request by vincentkam**
https://jira.mongodb.org/browse/CSHARP-2770

### CODE REVIEW

**Comment by vincentkam** [comment]
Ready for review!

File: build.cake [hunk 1]
```diff
@@ -90,16 +90,24 @@ Task("BuildArtifacts")
...
-                // DnsClient.dll is needed by Sandcastle
-                if (targetFramework == "net452" && project == "MongoDB.Driver.Core")
+                switch (project)
                 {
-                    fileNames.Add("DnsClient.dll");
+                    case "MongoDB.Driver":
+                         fileNames.Add("mongocrypt.dll");
+                         fileNames.Add("mongocrypt.pdb");
+                         break;
                 }
```
Comment (MikalaiMazurenka) [inline build.cake]: One case `switch` looks strange, it could be replaced with `if` safely

Comment (vincentkam) [reply build.cake]: This section has been removed since we no longer need to generate zip file releases per offline discussion :)

**Comment by vincentkam** [comment]
Sorry, requesting a pause in the review while @rstam takes over :)

**Review by DmitryLukyanov — APPROVED** [review]
Assuming that it was checked with `sandcastle` without any warnings/errors.
LGTM

**Comment by vincentkam** [comment]
Work done as part of: https://github.com/rstam/mongo-csharp-driver/pull/129

**Pull request closed without merging**